sechub-integrationtest/src/test/java/com/mercedesbenz/sechub/integrationtest/scenario2/UserChangeMailScenario2IntTest.java

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.integrationtest.scenario2;
+
+import static com.mercedesbenz.sechub.integrationtest.api.AssertMail.assertMailExists;
+import static com.mercedesbenz.sechub.integrationtest.api.TestAPI.*;
+import static com.mercedesbenz.sechub.integrationtest.api.TestAPI.getLinktToVerifyEmailAddressAfterChangeRequest;
+import static com.mercedesbenz.sechub.integrationtest.scenario2.Scenario2.USER_1;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.net.URI;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.mercedesbenz.sechub.integrationtest.api.IntegrationTestSetup;
+import com.mercedesbenz.sechub.integrationtest.api.TestUserDetailInformation;
+import com.mercedesbenz.sechub.integrationtest.api.TextSearchMode;
+
+public class UserChangeMailScenario2IntTest {
+
+    @Rule
+    public IntegrationTestSetup setup = IntegrationTestSetup.forScenario(Scenario2.class);
+
+    @Test
+    public void user_request_change_mail_address() {
+        /* prepare */
+        String email = USER_1.getEmail();
+        String newEmail = "new_user1@example.org";
+        assertTrue(email.endsWith("_user1@example.org"));
+
+        /* execute */
+        as(USER_1).requestChangeMailAddressTo(newEmail);
+
+        /* test */
+        String link = getLinktToVerifyEmailAddressAfterChangeRequest(newEmail, USER_1);
+        assertThat(link).isNotNull();
+
+        // workaround to avoid double encoding of the link
+        URI uri = null;
+        try {
+            uri = new URI(link);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Please don't do such things! (catch + printstacktrace + go further)

Why do we here a e.printStacktrace? It is also not clear why this is a workaround for what... 

Please add `@throws Exception` to the test method and just do Uri uri = new Uri(link). If there is an exception there is something odd and the test must fail.

Remark: It's possible to use junit5 extension `IntegrationTestExtension` instead of the old junit4 rules. Consider to use the new way.